### PR TITLE
network: e2e-net-test v419 skip should return 0

### PR DESCRIPTION
**Pull request by rafaeldtinoco**
## Description (git log)

... or else the v419 non CORE tests will fail.

Fixes: commit 8c67067b

Fixes: PR: #2456 (it had a bug)

## Type of change

- [ ] Bug fix (non-breaking change fixing an issue, preferable).
- [x] Quick fix (minor non-breaking change requiring no issue, use with care)
- [ ] Code refactor (code improvement and/or code removal)
- [ ] New feature (non-breaking change adding functionality).
- [ ] Breaking change (cause existing functionality not to work as expected).

### CODE REVIEW

File: tests/e2e-net-test.sh [hunk 1]
```diff
@@ -7,15 +7,14 @@
 error_info() {
     echo -n "INFO: "
     echo $@
-    exit 1
+    exit 0
```
Comment (yanivagman) [inline tests/e2e-net-test.sh:10]: This function is called error_info, so don't you want to return a value != 0

Comment (rafaeldtinoco) [reply tests/e2e-net-test.sh:10]: it should be called exit_info, fixing

Comment (rafaeldtinoco) [reply tests/e2e-net-test.sh:10]: fixed.

**Review by yanivagman — APPROVED** [review]
LGTM

**Pull request merged**